Assets/Prefabs/UI/InteractionText.prefab

```diff
@@ -75,7 +75,7 @@ MeshRenderer:
   m_AutoUVMaxAngle: 89
   m_LightmapParameters: {fileID: 0}
   m_SortingLayerID: -41943063
-  m_SortingLayer: 1
+  m_SortingLayer: 0
```

Comment:
is this to make sure it always renders in front?

Reply:
yeah 
its not actually required now 
i added a custom font for the interaction text but couldnt get it to render ontop even with this (and i had the distance shader on), so gave up.